pom.xml

```diff
@@ -6,7 +6,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>com.sonian</groupId>
   <artifactId>elasticsearch-jetty</artifactId>
-  <version>1.7.3-BV</version>
+  <version>1.7.3-SNAPSHOT</version>
```

Comment:
I think we should release a ES jetty version and have it here as well as polloi-ops.